pkg/skaffold/deploy/kustomize.go

```diff
@@ -118,6 +120,13 @@ func (k *KustomizeDeployer) Deploy(ctx context.Context, out io.Writer, builds []
 		return errors.Wrap(err, "setting labels in manifests")
 	}
 
+	for _, transform := range manifestTransforms {
```

Comment:
I assume these `manifestTransforms` are currently all `skaffold debug`-related transformers (because otherwise, it appears to me that this code change will affect other flows than `skaffold debug`.) That said, can this be any general transformers in this case later? If so, the error message "debug transform of manifests" may look out of place?

Reply:
Good point: although `debug` is the only user, it is a general facility. I'll revise it.

Reply:
I also forgot to include the `customization.yaml` file.